### ISSUE TITLE
I don't understand but i think this means it isn't detecting the media player??

### ISSUE THREAD

**Issue by CurlierSkate99** [aCluelessDanny/UeberPlayer#6]
![Screenshot 2020-08-17 at 8 59 15 PM](https://user-images.githubusercontent.com/20749897/90461792-dd5c8600-e0cc-11ea-826b-152c74bf687f.png)

**Comment by aCluelessDanny** [comment]
I'm not sure, but maybe I made a bad typo on one of the scripts?

I just put out a new release with some tweaks, so try reinstalling it with the latest version and check if that works out for you.

**Comment by jeffzma2000** [comment]
I'm still getting this error.

**Comment by aCluelessDanny** [comment]
Mmm, thing is that this error says that the script is just written incorrectly, so it's hard to say when the script itself should work.

There have been a few changes in the code since this issue's creation, so can you send me the error it shows now in the debug console just to be safe? Also which MacOS version are you using?

**Comment by IbrahimFadel** [comment]
I solved this by replacing all instances of "Music" with "iTunes" in ```lib/getTrack.scpt``` -- I guess the name changed since you made this.

**Comment by aCluelessDanny** [comment]
Oh heck, yeah that's definitely it. I changed it from "iTunes" to "Music" since forking the repo to test it on my Mac (I believe Catalina was the update that changed the app to "Music"?), but I completely forgot about making that compatible with iTunes again.

I should be able to make it compatible again without much issue

**Comment by IbrahimFadel** [comment]
I am on Mojave 10.14.6

Not sure when it changed though

**Comment by aCluelessDanny** [comment]
Yeah then it was Catalina since I'm on that, and it's the very next major MacOS version.

I just pushed out a new release that _in theory_ should fix this. I'm unable to test this fully on my Mac because it's already past that version, but it should work natively now from what I've gathered (though a tad hacky since it modifies the main script on widget init/refresh).

Do tell if it's working for ya now
